lib/modules/apostrophe-pages/lib/api.js

```diff
@@ -1183,14 +1183,9 @@ module.exports = function(self, options) {
       return setImmediate(callback);
     }
 
-    // Check if level is 0, then we are on the home page. Set the body class to
-    // apos-homepage so we can pick it up in JavaScript on the browser side.
+    // Add level as a data attribute on the body tag
     // The admin bar usses this to stay open if configured by the user
-    if (req.data.page && req.data.page.level === 0) {
-      self.apos.templates.addBodyDataAttribute(req, 'apos-home');
-    } else {
-      self.apos.templates.addBodyDataAttribute(req, 'apos-subpage');
-    }
+    self.apos.templates.addBodyDataAttribute(req, 'apos-level-' + req.data.page.level);
```

Comment:
@boutell I'm assuming here that `req.data.page.level` is **always** set here. I was scouting around the source to see if there's any indicator of that being a problem, but didn't find any such evidence. Should I add any safe checks here? If so please point me in the right direction of fallback because falling back to `0` is not right in my opinion if the value is undefined.

Reply:
Shouldn't the data attr eventually look like `data-apos-level="1"` rather than `data-apos-level-1`?

Or is it doing that and I'm misunderstanding

Reply:
I have no problem changing that, if that's the "house practice" :)